guardrails/src/drivers/rest/routers/admin.py

```
@@ -0,0 +1,270 @@
+        offset=result.offset,
+        limit=result.limit,
+    )
+
+
+# ==================== Activation ====================
+
+
+@router.post("/versions/{version_id}/activate", response_model=ActivateResponse)
+async def activate_version(
+    version_id: UUID,
+    uc: ActivateVersion = Depends(get_activate_version),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)